int32 colormap
---
int32 error